content/docs/1.0.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -75,6 +75,21 @@ This page covers the following topics:
 
 **Result:** Longhorn can store backups in S3. To create a backup, see [this section.](../create-a-backup)
 
+**Note:** If you operate Longhorn behind a proxy and you want to use AWS S3 as the backupstore, you must provide Longhorn information about your proxy in the `aws-secret` as below:
+    ```shell
+    AWS_ACCESS_KEY_ID: <your_aws_access_key_id>
+    AWS_SECRET_ACCESS_KEY: <your_aws_secret_access_key>
+    HTTP_PROXY: "http://proxy_ip:proxy_port"
```

Comment:
Those should be encoded in base64 in the secret right?

We can use the pattern above, e.g. Add the example of secret with comment.

Reply:
> e.g. Add the example of secret with comment.
I do not quite understand this. Pushed new changes. Is it what you mean?